core/blockchain.go

```diff
@@ -423,6 +435,9 @@ func NewBlockChain(db ethdb.Database, cacheConfig *CacheConfig, chainConfig *par
 			triedb.SaveCachePeriodically(bc.cacheConfig.TrieCleanJournal, bc.cacheConfig.TrieCleanRejournal, bc.quit)
 		}()
 	}
+
+	updateHeadL1msgGauge(bc.CurrentBlock())
```

Comment:
Would `CurrentBlock` be `nli`? seems no because checked the codebase and there are no `nil` checks when using it.

Reply:
thanks for the check.